scripts/create-keystore

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
Maybe the name of the script should be `create-android-keystore`? So people don't need to enter it to find out.